src/codemagic/tools/_app_store_connect/action_groups/beta_groups_action_group.py

```diff
@@ -0,0 +1,66 @@
+from typing import Sequence
+from typing import Union
+
+from codemagic import cli
+from codemagic.apple.app_store_connect.builds import Builds
+from codemagic.apple.app_store_connect.provisioning.beta_groups import BetaGroups
+from codemagic.apple.resources import Build
+from codemagic.apple.resources import ResourceId
+from codemagic.cli import Colors
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BuildArgument
+
+
+class BetaGroupsActionGroup(AbstractBaseAction):
+
+    @cli.action('modify-add-build',
```

Comment:
Naming again. I think `app-store-connect beta-groups add-build` already hints that the beta group is modified making the _modify_ part redundant.